flight/Libraries/inc/msplib.h

```diff
@@ -0,0 +1,225 @@
+/**
+ ******************************************************************************
+ * @addtogroup TauLabsLibraries Tau Labs Libraries
+ * @{
+ *
+ * @file       msplib.h
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2015-2016
```

Comment:
This should probably have a dronin copyright, shouldn't it?  Does it contain stuff developed at dronin that then moved to this file?

Otherwise, looks awesome, we're looking forward to picking this up.